src/main/resources/application.yaml

```
@@ -9,3 +9,8 @@ server:
 spring:
   mvc:
     log-resolved-exception: false
+  jpa:
+    open-in-view: true
+    properties:
+      hibernate:
+        default_batch_fetch_size: 100
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)